### PR TITLE
Chore: Create DeleteFile step

### PR DESCRIPTION
**Pull request by Skidaddle99**
I out-commented some code in other files that kept me from building, since the constructor is changed in the delete file step.. Tested the step using the install command and it works fine
That means that the actual changes are made in the DeleteFile.cs, and the rest are just outcommenting and code cleanup messing with git

### CODE REVIEW

File: src/UCommerce.SiteCore.Installer/Steps/DeleteFile.cs [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
 
-		public void Run(ITaskOutput output, NameValueCollection metaData)
-		{
-			_command.Delete(ex => new SitecoreInstallerLoggingService().Error<int>(ex));
-		}
-	}
-}
+        public async Task Run()
+        {
+            Console.WriteLine("Hej");
```
Comment (Victor-Allan) [inline src/UCommerce.SiteCore.Installer/Steps/DeleteFile.cs:21]: What's this for?

File: src/UCommerce.SiteCore.Installer/Steps/InstallStep.cs [hunk 2]
```diff
@@ -14,108 +15,107 @@ public InstallStep(string connectionString, string sitecorePath, bool backupDb,
...
-           //  steps.Add(new CreateSpeakApplicationIfSupported(sitecoreVersionChecker));
-           //
-           //  // Move sitecore config includes into the right path
-           //  ComposeMoveSitecoreConfigIncludes(sitecoreVersionChecker);
-           //  
-           //  // Clean up System.Collections.Immutable.dll in Lucene App since it is no longer used
-           //  steps.Add(new DeleteFile($"{virtualAppsPath}/Ucommerce.Search.Lucene/bin/System.Collections.Immutable.dll"));
-           //  steps.Add(new DeleteFile($"{virtualAppsPath}/Ucommerce.Search.Lucene.disabled/bin/System.Collections.Immutable.dll"));
-           //
+            steps.Add(new DeleteFile(new FileInfo("C:\\code\\hej.txt"), new SitecoreInstallerLoggingService()));
```
Comment (Victor-Allan) [inline src/UCommerce.SiteCore.Installer/Steps/InstallStep.cs:25]: Should this not be cleaned up? I assume this was used for testing

**Review by Victor-Allan — CHANGES_REQUESTED** [review]

File: src/UCommerce.SiteCore.Installer/Steps/DeleteFile.cs (hunk 1 above)
Comment (Skidaddle99) [reply src/UCommerce.SiteCore.Installer/Steps/DeleteFile.cs:21]: Good point

File: src/UCommerce.SiteCore.Installer/Steps/InstallStep.cs (hunk 2 above)
Comment (Skidaddle99) [reply src/UCommerce.SiteCore.Installer/Steps/InstallStep.cs:25]: Good point

**Review by Victor-Allan — APPROVED** [review]
Looks good

**Pull request merged**